NewEvaluation.css

```diff
@@ -68,3 +92,107 @@ body{
   	width:100%;  background-color: #dfdfdf; text-align: center; vertical-align: center; padding-bottom: 1em;
   	color:#828384;
 }
+
+
+.legend-box legend{
+	color:#03a8f3;
+	font-size: 18px;
+	font-weight: 500;
+	padding-left: 1em;
+	padding-right: 1em;
+}
+
+.legend-box li{
+	float:left;
+	width: 33%;
+	padding-bottom: 10px;
+}
+.legend-box select{
+	width: 40%;
+	padding: 3px;
+}
+.legend-box label{
+	display: inline-block;
+	width: 49%;
+}
+
+
+
+
+#butonu{
+	color:red;
+}
+
+#extra{
+	width: 22px;
+	float: left;
+	background-color: #03a8f3;
+}
+#top-list-radio{
+	list-style-type: none;
+	margin-top:0;
+	margin-bottom: 0;
+	color:white;
+}
+#top-list-radio li{
+	background-color: #03a8f3;
+	height: 4em;
+	line-height: 4em;
+	vertical-align: middle;
+
+}
+#down-list-radio{
+	list-style-type: none;
+	margin-top:0;
+	padding:10px 0px 10px 0px;
+	margin-bottom: 0;
+	border:1px solid black;
+}
+#down-list-radio li{
+	padding-right:10px;
+	float:left;
+}
+
+#placeHolder{
+	width: 50px;
+	float: left;
+}
+#RR1{
+
+	width: 13%;
+	float: left;
+	padding-left: 60px;
+}
+.spacer{
+	width: 20%;
+}
+
+.clearfix{
+	clear:both;
+}
+#R1{
+	width: 16%;
+	float: left;
+}
+.R2{
+	width: 26%;
+	float: left;
+
+}
+
+input[type=submit]{
```

Comment:
I think this takes all input tab and check whethere the type of the input is of type "submit". I used this because I had problems with adding CSS classes to the part I used it for.